specification/behaviors.md

```diff
@@ -0,0 +1,50 @@
+# Behaviors
+
+## Profiling
+
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+This document describes the specific behavior and conventions for Splunk
+instrumentation libraries that contain profiling features.
+
+### Call Stack Sampling
+
+An instrumentation library that has profiling capabilities MUST be able to
+sample call stacks at a fixed interval. This default interval MUST default to 1
+second.
+
+When a language runtime supports threading, stacks MUST be sampled across all
```

Comment:
unless we mention here that there is an exception described below in "filtering", we have a contradiction. "MUST be sampled across all threads" and "some threads SHOULD NOT be sampled"

Reply:
This is a fair point, but I see the filtering as happening downstream of the sampling. The actual sampling mechanism may be more difficult to control (such as in the case of Java, where we can't control which threads JFR reports on). I don't think it's a contradiction.